src/constants/config.json

```diff
@@ -5,10 +5,14 @@
     "dataLayerIndex": 1
   },
   "eduAPI": {
-    "authToken": "64aa60e488a555e2ef8b7f9d1621ec7e430dcb8b8baa0d00b8460eca5660d188",
+    "testing-only-authToken": "64aa60e488a555e2ef8b7f9d1621ec7e430dcb8b8baa0d00b8460eca5660d188",
     "root": "https://education.staging.zooniverse.org/",
     "students": "students/classrooms/",
     "teachers": "teachers/classrooms/"
   },
+  "routes": {
```

Comment:
It's a coincidence that the routes are similar. But I wanted to make clear that `routes` is for internal, app use, while `eduAPI` is used for external, API use.